### ISSUE TITLE
problems with pdflatex

### ISSUE THREAD

**Issue by halpern** [MiKTeX/miktex#1087]
- [x] I have read and acknowledged this HOWTO article:  
  [Submitting a MiKTeX problem report](https://miktex.org/howto/bug-report)

I have a macbook pro with an M1 chip.  All of a sudden pdflatex stopped working for me today.  (It's been working fine ever since I got the machine about a year ago.   I've had other problems today with emacs, which I suspect are related.  In any case,
I'm appending the relevant parts of all the log files that have changed:

(1) Here is the the relevant part of /Users/joehalpern/Library/Application Support/MiKTeX/texmfs/data/miktex/log/miktex.log:

2022-04-09 14:43:44,023-0400 INFO  miktex - this is One MiKTeX Utility 1.0 (MiKTeX 22.1)
2022-04-09 14:43:44,023-0400 INFO  miktex - this process (5857) started by miktex-pdftex in directory /Users/joehalpern/Documents/work with command line: miktex formats build --key pdflatex --engine pdftex
2022-04-09 14:43:44,025-0400 INFO  miktex - Building format 'pdflatex' with engine 'pdftex'...
2022-04-09 14:43:44,070-0400 INFO  miktex.core - forking...
2022-04-09 14:43:44,424-0400 ERROR miktex - sub-process error output has been saved to '/Users/joehalpern/Library/Application Support/MiKTeX/texmfs/data/miktex/log/miktex-makefmt_2022-04-09-144344.out'
2022-04-09 14:43:44,425-0400 FATAL miktex - Not a directory: path="/Users/joehalpern/Documents/work/tex/latex"
2022-04-09 14:43:44,426-0400 FATAL miktex - Info: path="/Users/joehalpern/Documents/work/tex/latex"
2022-04-09 14:43:44,426-0400 FATAL miktex - Source: Libraries/MiKTeX/Core/Directory/unx/unxDirectory.cpp
2022-04-09 14:43:44,426-0400 FATAL miktex - Line: 73
2022-04-09 14:43:44,426-0400 INFO  miktex - this process (5857) finishes with exit code 1

(2) In case it helps, here is the relevant part of Users/joehalpern/Library/Application Support/MiKTeX/texmfs/data/miktex/log/pdflatex.log, which is largely redundant:

2022-04-09 14:43:43,692-0400 INFO  pdflatex - this is MiKTeX-PDFTEX 4.10.0 (1.40.24) (MiKTeX 22.1)
2022-04-09 14:43:43,693-0400 INFO  pdflatex - allowing known shell commands
2022-04-09 14:43:43,699-0400 INFO  pdflatex - going to create file: pdflatex.fmt
2022-04-09 14:43:43,744-0400 INFO  pdflatex.core - forking...
2022-04-09 14:43:44,428-0400 ERROR pdflatex - /Applications/MiKTeX Console.app/Contents/bin/miktex did not succeed; exitCode: 1
2022-04-09 14:43:44,428-0400 ERROR pdflatex - output:
2022-04-09 14:43:44,428-0400 ERROR pdflatex - 
Sorry, but: Not a directory: path="/Users/joehalpern/Documents/work/tex/latex"

2022-04-09 14:43:44,428-0400 FATAL pdflatex.core - The memory dump file could not be found.
2022-04-09 14:43:44,428-0400 FATAL pdflatex.core - Data: fileName="pdflatex.fmt"
2022-04-09 14:43:44,428-0400 FATAL pdflatex.core - Source: Libraries/MiKTeX/TeXAndFriends/texmfapp.cpp:717
2022-04-09 14:43:44,429-0400 FATAL pdflatex - The memory dump file could not be found.
2022-04-09 14:43:44,429-0400 FATAL pdflatex - Info: fileName="pdflatex.fmt"
2022-04-09 14:43:44,429-0400 FATAL pdflatex - Source: Libraries/MiKTeX/TeXAndFriends/texmfapp.cpp
2022-04-09 14:43:44,429-0400 FATAL pdflatex - Line: 717
2022-04-09 14:43:44,430-0400 INFO  pdflatex - this process (5856) finishes with exit code 1

(3) Here is the relevant part of mkfntmap.log

2022-04-09 12:55:13,105-0400 INFO  mkfntmap - this process (87352) started by 'initexmf' with command line: mkfntmap --miktex-disable-maintenance --miktex-disable-diagnose
2022-04-09 12:55:13,106-0400 INFO  mkfntmap - Parsing config file "/Users/joehalpern/Library/Application Support/MiKTeX/texmfs/install/miktex/config/updmap.cfg"...
2022-04-09 12:55:13,107-0400 FATAL mkfntmap - invalid configuration setting
2022-04-09 12:55:13,107-0400 FATAL mkfntmap - cfg file: /Users/joehalpern/Library/Application Support/MiKTeX/texmfs/install/miktex/config/updmap.cfg
2022-04-09 12:55:13,107-0400 FATAL mkfntmap - line: 87

2022-04-09 12:55:13,105-0400 INFO  mkfntmap - this process (87352) started by 'initexmf' with command line: mkfntmap --miktex-disable-maintenance --miktex-disable-diagnose
2022-04-09 12:55:13,106-0400 INFO  mkfntmap - Parsing config file "/Users/joehalpern/Library/Application Support/MiKTeX/texmfs/install/miktex/config/updmap.cfg"...
2022-04-09 12:55:13,107-0400 FATAL mkfntmap - invalid configuration setting
2022-04-09 12:55:13,107-0400 FATAL mkfntmap - cfg file: /Users/joehalpern/Library/Application Support/MiKTeX/texmfs/install/miktex/config/updmap.cfg
2022-04-09 12:55:13,107-0400 FATAL mkfntmap - line: 87

Let me know if you need further information.  If you want to contact me directly, my email is halpern@cs.cornell.edu

Thanks,
Joe Halpern

**Comment by edocevoli** [comment]
Directory `/Users/joehalpern/Documents/work/tex/latex` does not exist or cannot be accessed by MiKTeX. Please fix this in order to get MiKTeX going again.

**Comment by halpern** [comment]
Thanks for your help!  I removed /Users/joehklpaern/Documents/work from my paths on MikTex  and that solved the problem.  But (1) the directory been there all along and hasn't caused a problem; the problem just started yesterday; (2) I have an older Mac Air with the same MikTex paths and there isn't a problem on that, (3) That path is the one where I store all the files that I latex and many style files, which is why I wanted it on the path, (4) Interestingly, when I tried adding that directory back to the list of search paths after removing it, it wouldn't allow me to do it (or to change the order of paths), giving me the error "Not a directory: path="/Users/joehalpern/Documents/work/tex/generic".  As I said, that directory has lots of style files that I use.  On the other hand, that's the directory I'm typically n when I use pdflatex, which perhaps means it's not a problem.  Any thoughts?
